src/boot/driverominit.c

```diff
@@ -1,5 +1,9 @@
 #include "global.h"
 
+extern OSPiHandle* __osCurrentHandle[];
+extern void __osPiGetAccess(void);
+extern void __osPiRelAccess(void);
```

Comment:
Personally I'm not a huge fan of duplicating declarations instead of including a header declaring them.
I think it is fine for this file to include a private ultralib header since it is actually accessing private functions

Reply:
Also, why the `extern`s on the function declarations?

Reply:
I originally thought it wasn't kosher for OOT to access "interal" libultra headers, but after talking with ultralib folks it seems okay so I've removed these.

The `extern` was to make it clear that the function was not defined in the compilation unit, since most of the time in OOT it will be